frontend/rust-lib/flowy-document2/src/parser/parser_entities.rs

```diff
@@ -479,3 +454,51 @@ impl NestedBlock {
     text
   }
 }
+
+pub struct ConvertBlockToHtmlParams {
+  pub prev_block_ty: Option<String>,
+  pub next_block_ty: Option<String>,
+}
+
+#[derive(PartialEq, Eq, Debug, ProtoBuf_Enum, Clone, Default)]
+pub enum InputType {
+  #[default]
+  Html = 0,
+  PlainText = 1,
+}
+
+#[derive(Default, ProtoBuf, Debug)]
+pub struct ConvertDataToJsonPayloadPB {
```

Comment:
You can try this.
<img width="639" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/86001920/eeb93936-b799-4e65-9794-6c330e392524">
<img width="863" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/86001920/e4e6ba28-0ec9-4390-80bb-f94b9f831628">